src/server.h

```diff
@@ -2058,6 +2058,7 @@ struct valkeyServer {
         int dbid;
     } repl_provisional_primary;
     client *cached_primary;             /* Cached primary to be reused for PSYNC. */
+    rio *loading_rio;                   /* Pointer to the rio object currently used for loading data. */
```

Comment:
Shouldn't we make sure to first nullify it in initServerConfig?

Reply:
right